src/Makefile.am

```diff
@@ -170,12 +172,12 @@ sphthread_t_LDADD   = libsphde.la
 TESTS                  += sphlflogger_tt
 sphlflogger_tt_SOURCES  = tests/sphlflogger_tt.c
 sphlflogger_tt_CFLAGS   = -DSPH_TIMERTEST_VERIFY=1
-sphlflogger_tt_LDADD    = libsphde.la
+sphlflogger_tt_LDADD    = libsphde.la -litm
```

Comment:
I confess I'm not completely sure why, but this test (and the one below) both failed to link without -litm.  I can't see any dependencies which would cause this. Hmm, I just tried without these added libraries, and the link succeeds. I presume they can be removed. Sorry for the confusion.